### ISSUE TITLE
CNN from scratch

### ISSUE THREAD

**Issue by PyAllegro** [dasoto/skincancer#2]
![acc](https://user-images.githubusercontent.com/38966664/47960039-4a08ac80-e02e-11e8-86f1-a5ab145f6bfa.png)
![loss](https://user-images.githubusercontent.com/38966664/47960040-4aa14300-e02e-11e8-861b-c2b09da4e9b0.png)
When I tried to train with your training script (train_model. new. py), I found that ACC curve and loss performed very badly. What might be the reason? Is my way of data preprocessing wrong?
Thank you very much for any help.

**Comment by Alramzey** [comment]
I have the same problem except my training and validation accuracy are oscillating from 53 to ~63, did you figure out what was the issue?
